### PR TITLE
Nerfs the Robot VTEC Module

### PR DESCRIPTION
**Pull request by Frenjo**
Reduces the movement delay reduction of the Robot VTEC Module from 1 -> 0.5 as it's currently so fast as to be unplayable.

For coders: Also does some prettification and cleanup of the robot_upgrades.dm file, see commit log for details.

### CODE REVIEW

**Comment by Spookerton** [comment]
@Frenjo For the future, please split functional changes and purely stylistic style changes into separate commits - it makes hunting down what's truly changed a great deal easier.

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -1,67 +1,81 @@
...
 	R.name = heldname
 	R.custom_name = heldname
 	R.real_name = heldname
+	return TRUE
 
-	return 1
-
+/*
+ * Restart Module
+ */
```
Comment (Spookerton) [inline code/game/objects/items/robot/robot_upgrades.dm:66]: I won't ask you to remove them, but in my opinion none of these need a comment saying what's said two lines below.

Aside, avoid using block comments for trivial things. If you want visually better-separated concerns, double newline between them and be cagey about adding any singles as well.

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 2]
```diff
@@ -70,72 +84,79 @@
...
 	living_mob_list |= R
 	R.notify_ai(ROBOT_NOTIFICATION_NEW_UNIT)
-	return 1
-
+	return TRUE
 
+/*
+ * VTEC Module
+ */
+#define VTEC_MOVE_DELAY_REDUCTION 0.5
```
Comment (Spookerton) [inline code/game/objects/items/robot/robot_upgrades.dm:92]: This is needlessly noisy. Change back to in-line number literals or add `var/move_delay_reduction = 0.5` to the type.

**Review by Spookerton — CHANGES_REQUESTED** [review]

**Review by Spookerton — APPROVED** [review]

File: code/game/objects/items/robot/robot_upgrades.dm (hunk 1 above)
Comment (Atermonera) [reply code/game/objects/items/robot/robot_upgrades.dm:66]: But I will!
Files that get too long are a pet peeve of mine, and needless comments (especially needless block comments) are a very good way to reach that state.

Comments should explain concisely what the code cannot. Restating the obvious is wasteful.
Within the scope of ss13 classes, either the type or the name or the description should plainly describe what a thing is. File purpose should be encapsulated by the filename, but I'm not so hostile towards file headers.

**Review by Atermonera — CHANGES_REQUESTED** [review]

**Review by Atermonera — APPROVED** [review]

**Pull request merged**